timemachine/fe/bar.py

```diff
@@ -235,7 +204,8 @@ def df_from_ukln_by_lambda(ukln_by_lambda: NDArray) -> Tuple[float, float]:
     for lambda_idx in range(ukln_by_lambda.shape[0]):
         window_ukln = ukln_by_lambda[lambda_idx]
 
-        dF, dF_err = df_from_ukln(window_ukln)
+        w_fwd, w_rev = works_from_ukln(window_ukln)
+        dF, dF_err = pymbar.BAR(w_fwd, w_rev)
```

Comment:
nit: units: are the outputs assumed in kBT or kJ/mol?

Reply:
Outputs are in kT. (The only change here was to pull out a helper function to compute forward and reverse works from the reduced energies $u_{k l n}$, and then eliminate `df_from_ukln` and `df_err_from_ukln`, which just become 2-liners). It didn't seem necessary to state the output units explicitly since this seems to be the convention for other functions in this module. Do you think we should update this and the others?

Reply:
Sorry, this was a tangent -- not a change introduced by this PR, but just showed up in the diff.

(convention: I prefer to use lower case for reduced quantities `u`, `w`, `df` (in kBT), uppercase `U`, `W`, `dF` (in kJ/mol). The code near this diff already mixed these conventions.)

Reply:
Ah, I see. I agree with the preference for lower-case here, but would prefer to change it in a separate PR as it would introduce a fair amount of noise to the diff here.